sympy/assumptions/tests/test_query.py

```diff
@@ -1523,6 +1523,10 @@ def test_integer():
     assert ask(Q.integer(x/2), Q.even(x)) is True
     assert ask(Q.integer(x/3), Q.odd(x)) is None
     assert ask(Q.integer(x/3), Q.even(x)) is None
+    assert ask(Q.integer(x/5), Q.odd(x)) is None
+    assert ask(Q.integer(x/5), Q.even(x)) is None
+    assert ask(Q.integer(x/7), Q.odd(x)) is None
+    assert ask(Q.integer(x/7), Q.even(x)) is None
```

Comment:
Are these interesting?

Reply:
Yup, these cases need cleaning. Working on it.

Reply:
Done